connectors/src/connectors/intercom/temporal/sync_conversation.ts

```diff
@@ -0,0 +1,218 @@
+import type { ModelId } from "@dust-tt/types";
```

Comment:
Nit: should this file really be in the `temporal` subdirectory ? My impression is that they are rather Intercom "lib" functions that are called from activities (in activities.ts) but are not activities themselves

Reply:
That's right! I'm happy to move it, but as I have 2 stacked PR on top of this one I'm going to do it right after merging them all avoid an annoying rebase 🙈